### PR TITLE
Main Ping Exclusion Zone Cleanup

### PR DESCRIPTION
**Pull request by fbertsch**
ChildPayloads and Log were removed in 61.

Checklist for reviewer:

- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [ ] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: templates/include/telemetry/mainPayload.1.schema.json [hunk 1]
```diff
@@ -92,18 +67,17 @@
...
       "items": {
         "type": "object",
-        "properties": { }
+        "properties": {
+          "type": {"type": "string"},
+          "action": {"type": "string"},
+          "method": {"type": "string"},
+          "extras": {"type": "string"},
+          "timestamp": {"type": "number"}
+        }
```
Comment (jklukas) [inline templates/include/telemetry/mainPayload.1.schema.json]: This looks like an addition rather than a subtraction. Is this an unrelated change, or does this directly support the exclusions mentioned in the description?

Also, is this giving names to the elements of the event array? Is this a solution for letting the jsonschema-transpiler output named structs for tuples?

Comment (fbertsch) [reply templates/include/telemetry/mainPayload.1.schema.json]: This is an addition, however this is an event _object_, not an array, so it does not solve the tuple issue. (Yes is not technically an exclusion but still needed for production :))

**Comment by fbertsch** [comment]
@acmiyaguchi are you looking into the missing integration report tests?

**Comment by acmiyaguchi** [comment]
@fbertsch Yes, I'm currently taking a look at it, it's not immediately obvious why it's spitting out empty files.

**Comment by fbertsch** [comment]
Great. Mark and I just confirmed that these fields are _not_ in the schema so this change is still good to merge.

**Comment by mreid-moz** [comment]
See [this comment](https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/357#issuecomment-513919393) for info on the integration test fix.

**Comment by fbertsch** [comment]
Okay, found the diff. I'm not sure if those `saved-session` errors are new.
```
        "telemetry.main.4": {
            "error_count": 0,
            "error_rate": 0.0,
            "errors": null,
            "time": 2.47,
            "total": 1000
        },
        "telemetry.saved-session.4": {
            "error_count": 22,
            "error_rate": 2.2,
            "errors": {
                "Validation Error: ('type', '#/properties/clientId', '#/clientId')": 3,
                "Validation Error: ('type', '#/properties/environment/properties/build/properties/vendor', '#/environment/build/vendor')": 19
            },
            "time": 6.31,
            "total": 1000
        }
```

**Comment by fbertsch** [comment]
@mreid-moz RFAL

**Comment by mreid-moz** [comment]
I re-ran the integration test and the main ping error rate did not change.

**Comment by fbertsch** [comment]
Great, thanks!
